ui/book/handlers.py

```diff
@@ -16,6 +16,7 @@
 
 
 async def init(book):
+    manual = Manual()
     if book.filename == manual.filename:
```

Comment:
You can just import `manual_filename` and use that instead of creating new instances.